vitest.config.ts

```diff
@@ -2,7 +2,10 @@ import { defineConfig } from 'vitest/config'
 
 export default defineConfig({
   test: {
-    environment: 'happy-dom',
+    environment: 'node',
```

Comment:
Rejig to tests to try to automatically use the correct environment by default.

(Can also use `// @vitest-environment` comments in individual files to override.)